package.json

```
@@ -40,7 +40,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
-    "customerio-reactnative": "*"
+    "customerio-reactnative": ">=1.0.1-alpha.3"
```

The version requirement should be: 
`>= 1.0.1, < 2.0` This code as it is in this PR, we will install v2 which we don't want to do. We may need to migrate our expo plugin for major versions so we don't want to install versions automatically. 

That's why I suggest: 

```suggestion
    "customerio-reactnative": "^1.0.0"
```